src/Nordea/Components/Error.elm

```diff
@@ -52,38 +53,33 @@ type Error msg
     = Error ErrorConfig
 
 
-internalServerError : Translation -> Error msg
-internalServerError translate =
+internalServerError : Translation -> String -> Error msg
+internalServerError translate supportEmail =
     init
         { errorType = InternalServerError
-        , customSupportEmail = Nothing
+        , supportEmail = Just supportEmail
```

Comment:
Would it be possible to model ErrorConfig to be either a InternalServerErrorConfig or a PageNotFoundErrorConfig, so we don't need to have supportEmail as a Maybe field? It would only be available to InternalServerErrorConfig

Reply:
Any suggestions on how to do that in a smooth way?

Reply:
Good idea! But in that case, if the configs get very different, would it make sense to just split them completely? Separate components for internal server error and page not found? They could still live in the same file though, and share sub views if needed.

Reply:
Something like this maybe? 🤔

`type ErrorConfig
    = InternalServerErrorConfig
        { errorType : ErrorType
        , supportEmail : Maybe String
        , translate : Translation
        }
    | PageNotFoundErrorConfig
        { errorType : ErrorType
        , translate : Translation
        }`

Reply:
With that, the `ErrorType` is redundant and can be removed 😀

Reply:
I really like the idea of two separate views, since we have this viewAction thing that is not supposed to be shown on PageNotFoundError anyway. I will try to do some changes here 😊 I'll come back to you!